src/common/Collision/Management/VMapManager2.cpp

```diff
@@ -279,7 +280,7 @@ namespace VMAP
         return false;
     }
 
-    WorldModel* VMapManager2::acquireModelInstance(const std::string& basepath, const std::string& filename)
+    WorldModel* VMapManager2::acquireModelInstance(const std::string& basepath, const std::string& filename, uint32 flags/* Only used when creating the model */)
```

Comment:
Yea, I agree. I think in general that method shouldn't be both used for loading and creating.

However, since this was how the accepted and merged commit you linked me the other day [had it](https://github.com/TrinityCore/TrinityCore/commit/c53d722aa29ff3a39d881de7d6ef86b79deffe48) I left it as is.

It has a default defined at least. I made a note that I didn't fully agree with how the originally commit was implemented, I believe it exposed a public field too, but I didn't want to make changes to it since it had actually been accepted in the past that way.